### ISSUE TITLE
Most channels that loads with the malay subs cannot load

### ISSUE THREAD

**Issue by greknic** [pcccw/pcccw.github.io#9]
Hi there,

Most channels that loads with the malay subs cannot load.
Kept showing "AnErrorOccured:DRMSessionException"

**Comment by greknic** [comment]
hi pcccw, is not astro channel cannot load. i meant that most channels that uses malay subtitles does not load (those that is tapped from indonesia source), like Fox, HGTV, Celestial, Wakuwaku japan, etc. 
It shows the error "AnErrorOccured:DRMSessionException"

**Comment by pcccw** [comment]
Yes, those channel are down as the provider has changed the drm key to the channels.

I love those channels but I am not mnc.
